gradle.properties

```diff
@@ -21,3 +21,4 @@ buildType=BUILD
 org.gradle.caching=true
 org.gradle.parallel=true
 org.gradle.logging.level=warn
+systemProp.file.encoding=utf-8
```

Comment:
Why? Usually you set Gradle properties in gradle.properties... I don't think this will have your intended outcome (that Gradle is handling everything in UTF-8)...